spec/overcommit/hook/pre_commit/yaml_lint_spec.rb

```diff
@@ -34,9 +34,22 @@
       )
     end
 
-    it { should warn }
+    it { should fail }
```

Comment:
Note that `fail` is a keyword in Ruby so this test will always error (see the failing TravisCI run).

You should use the `fail_hook` keyword here and below.

Reply:
Ok, it is done. I admit ruby is not my main language. I took inspiration from the existing code in this matter.